packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -58,19 +60,19 @@ class SystemRulesTable extends React.Component {
     }
 
     setInitialCurrentRows() {
-        const { hidePassed, itemsPerPage, severity, policy } = this.state;
+        const { hidePassed, itemsPerPage, severity, policy, selectedFilter } = this.state;
         let { profileRules } = this.props;
         const rowsRefIds = this.rulesToRows(profileRules);
 
         this.setState({
-            currentRows: this.currentRows(1, itemsPerPage, rowsRefIds),
+            currentRows: this.markSelectedRows(this.currentRows(1, itemsPerPage, rowsRefIds)),
             originalRows: rowsRefIds.rows,
-            rows: rowsRefIds.rows,
+            rows: this.markSelectedRows(rowsRefIds.rows),
             profiles: rowsRefIds.profiles,
             refIds: rowsRefIds.refIds
         }, () => {
-            if (hidePassed) {
-                this.updateFilter(hidePassed, severity, policy);
+            if (hidePassed || selectedFilter) {
+                this.updateFilter(hidePassed, severity, policy, selectedFilter);
```

Comment:
If we pass the `selectedFilter` enabled from the create policy wizard, all of the pre-selected rules will show up as selected